### PR TITLE
Issue/browse card formatting

### LINKED ISSUE #93: Browse Card Quick fixes
## Description
Quick updates to the browse card

### Acceptance Criteria
1. Alpha sorting by university name
2. Formatting of student population to be integers and commas
3. Cost per year to have $$ and comma
4. Graduation rate to be formatted as percentage
5. Change student population to undergraduate population
6. Remove filtering until we build filtering

### Resources:
image

## Definition of Done
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)

### PR DESCRIPTION
**Pull request by TheDanMiller**
# Some formatting quick fixes

# Summary
Formatting for card, closes #93

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How To Test
1. Pull down code
2. Look at prettier numbers

# Checklist:

- [x] My code follows the style guidelines of this project and has no linting errors
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, including any applicable ADRs
- [x] My changes generate no new warnings
- [x] I have added tests that fail without these changes
- [x] New and existing tests (unit, integration, accessibility) pass locally
- [x] Documentation updated
- [x] If there are security concerns they are addressed or ticketed after being discussed

### CODE REVIEW

File: src/app/components/cards/CollegeCard.tsx [hunk 1]
```diff
@@ -9,8 +9,12 @@ import {
   CardMedia,
 } from "@trussworks/react-uswds";
 import { CardIcon } from "./CardIcon";
-// types
+//types
```
Comment (gmrabian) [inline src/app/components/cards/CollegeCard.tsx:12]: nooooooooooo

File: src/app/utils/masking.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * Converts a float number to a percentage string
+ * @param {Number} value
+ * @returns {string}
+ */
+export function convertPercentage(value: number): string {
+  let maskValue = value * 100;
+  maskValue = Math.round(maskValue);
+
+  return maskValue.toString();
+}
```
Comment (gmrabian) [inline src/app/utils/masking.ts:23]: if we're converting to a string is it worth adding the percentage sign % here as well?

**Review by gmrabian — APPROVED** [review]

File: src/app/utils/masking.ts (hunk 2 above)
Comment (TheDanMiller) [reply src/app/utils/masking.ts:23]: I thought about it, same with the dollar sign, but I felt like that muddied the purpose of the function

File: src/app/components/cards/CollegeCard.tsx (hunk 1 above)
Comment (TheDanMiller) [reply src/app/components/cards/CollegeCard.tsx:13]: Are my setting wrong?

**Review by braxex — APPROVED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**